packages/csp/src/evaluator.js

```diff
@@ -28,11 +28,18 @@ function generateEvaluator(el, expression, dataStack) {
     return (receiver = () => {}, { scope = {}, params = [] } = {}) => {
         let completeScope = mergeProxies([scope, ...dataStack])
 
-        if (completeScope[expression] === undefined) {
-            throwExpressionError(el, expression)
-        }
-
-        runIfTypeOfFunction(receiver, completeScope[expression], completeScope, params)
+        let evaluatedExpression = expression.split('.').reduce(
+            (currentScope, currentExpression) => {
+                if (currentScope[currentExpression] === undefined) {
```

Comment:
```suggestion
                if (!currentExpression in currentScope) {
```

should help in scenarios where the thing is expected to be undefined.

But also, this will now exhibit a different behavior than non-csp in regards to this, where it would only error when trying to access a key on the thing that is undefined, but would be happy to return undefined.

Reply:
Good point, I kept the same check as the original version. Happy to change it if Caleb is cool with that.